papermill/tests/test_iorw.py

```diff
@@ -48,6 +48,10 @@ def __init__(self, ver):
             self.ver = ver
 
         def read(self, path):
+            if path == 'fake/path_bytes':
```

Comment:
Let's not make this so complicated in the test (plus this isn't testing that the code in iorw module actually does the decode you repeat here). Instead make a FakeByteHandler class which reads and returns the notebook you attached above. Then make a special test for reading from that handler and proving it does not raise an exception. This will test your code path cleanly.

Reply:
Thanks for the review, submitted new test, PTAL.